templates/commands/render/render.go

```diff
@@ -508,6 +548,17 @@ func executeSteps(ctx context.Context, steps []*spec.Step, sp *stepParams) error
 			logger.WarnContext(ctx, contents)
 		}
 	}
+
+	if sp.flags.DebugStepDiffs {
+		// Use default log level.
+		logger.WarnContext(
+			ctx,
+			fmt.Sprintf(
+				"Please navigate to '%s' or use 'git --git-dir=%s <command>' to see commits/diffs for each step",
+				sp.debugDir, sp.debugDir,
```

Comment:
Should we include the scratch dir in this message? I'm not sure, but I think the user will need to cd into the scratch dir and use --git-dir=debugDir. A full working example command might be helpful (perhaps `git log`).

Reply:
Will add a full command and in readme too, scratch dir is already removed though with only debug-step-diffs flag, do you mean to have --keep-temp-dirs enabled too when --debug-step-diffs is enabled which might cause confusions though.

Reply:
Oh right, I see what you mean about the scratch dir. So let's just include a full command in this log message, like you said.

Reply:
For your concern, I tested this flag locally, it does not require me to cd into the scratch dir to use --git-dir flag.

Reply:
SGTM.